TSPL.docc/footer.html

```diff
@@ -110,6 +110,13 @@
         padding: 0;
         border:0
     }
+
+    .footer-social-icons {
+      display: flex;
+      flex-direction: row-reverse;
+      justify-content: center;
```

Comment:
```suggestion
      justify-content: right;
```

Right justification would better match how these icons appear on the main swift.org footer.